lib/kernel/doc/src/logger_std_h.xml

```diff
@@ -55,7 +55,7 @@
        is stored in a sub map with the key <c>config</c>, and can contain the
        following parameters:</p>
     <taglist>
-      <tag><marker id="type"/><c>type = standard_io | standard_error | file</c></tag>
+      <tag><marker id="type"/><c>type = standard_io | standard_error | file | {device, atom() | io:device()}</c></tag>
```

Comment:
`io:device()` already includes `atom()`, so no need to include it in the list.

Reply:
Now it should be fixed.